Removing RunTime Postprocessor

permcody opened the pull request:
closes #8765

I really don't think we need to deprecate this thing. I don't think anyone really uses it in their test suite or input files. Let's see what CIVET says...

permcody committed:
Removing RunTime Postprocessor

closes #8765

moosebuild commented:
Job `Test intel:linux-intel` on 725a720 : invalidated by @permcody

Intel is a steaming pile

moosebuild commented:
Job `Moose Documentation:linux-clang` on 725a720 wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/9369/site

permcody committed:
Remove MooseDoc stubs

refs #8765

moosebuild commented:
Job `Moose Documentation:linux-clang` on 42cf904 wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/9369/site

dschwen commented:
> I really don't think we need to deprecate this thing. I don't think anyone really uses it in their test suite or input files. 

Fails 22 bison tests

permcody commented:
Examples, not tests. I can switch those over to the other, better object
before this is merged.
On Fri, Jun 23, 2017 at 10:30 AM Daniel Schwen <notifications@github.com>
wrote:

> I really don't think we need to deprecate this thing. I don't think anyone
> really uses it in their test suite or input files.
>
> Fails 22 bison tests
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/idaholab/moose/pull/9369#issuecomment-310712537>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AC5XID8b6VrU-4pLEn27oi6L25nUv2IBks5sG-g-gaJpZM4OCzqY>
> .
>

permcody committed:
Improve usability of PerformanceData Postprocessor

Adding a couple expandable MooseEnums to assist users with picking the correct
options when using the Postprocessor. This also allows for case-insensitive
matches when using "alive" or "active".

refs #8765

moosebuild commented:
Job `Moose Documentation:linux-clang` on 35dbfd3 wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/9369/site

moosebuild commented:
Job `Moose Documentation:linux-clang` on cf19462 wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/9369/site

moosebuild commented:
Job `Moose Documentation:linux-clang` on c2931da wanted to post the following:

View the site at http://mooseframework.org/docs/PRs/9369/site

andrsd commented on test/tests/postprocessors/misc_pps/misc_pps_test.i:
```diff
@@ -14,16 +14,13 @@
 [AuxVariables]
   [./aux_var]
     family = SCALAR
+    initial_condition = 42
```
Why was this necessary?

permcody replied:
There's a ScalarVariable PP below that reports the value of the ScalarVariable. That's actually part of what this test is supposed to do, it shows that we can output several different Postprocessors to the Exodus file. It's actually a pretty crappy test, but that's what it does. 

The reporter was correctly reporting zero because there are no ScalarKernels in this input file but I figured that was boring. Since I had to regold it anyway, I figured I'd put a value in there so that we could "prove" that it was working properly. As you'd expect the Scalar value is correct in the initial condition and the report lags one timestep (TIMESTEP_END).

andrsd replied:
Ok, I thought you were just removing code in this patch, so it looked like a left over from something else...
